pelican/__init__.py

```diff
@@ -527,6 +533,23 @@ def main():
 
     try:
         pelican, settings = get_instance(args)
+
+        if args.print_settings:
+            from pprint import pprint
```

Comment:
Is there a reason not to have this at the top of the file with the other import statements?

Reply:
No reason. I agree it's better at top. Fixed. Thanks!